linode/helper/normalize_test.go

```diff
@@ -0,0 +1,33 @@
+package helper_test
+
+import (
+	"testing"
+
+	"github.com/linode/terraform-provider-linode/linode/helper"
+)
+
+func TestCompareIPv6Ranges(t *testing.T) {
+	ips := []string{
+		"1111:1111::1111:1111:1111:f88/128",
+		"1111:1111::1111:1111:1111:0f88/128",
+	}
+
+	result, err := helper.CompareIPv6Ranges(ips[0], ips[1])
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !result {
+		t.Fatalf("ranges are reported as different despite being equal")
+	}
+
+	ips[1] = "1111:1111::1111:1211:1111:0f88/126"
```

Comment:
shouldn't this be a /128?

Reply:
For this part of the test I made a few arbitrary changes to test that comparing unequal IPv6 ranges will return false. It's probably a good idea to test mask and IP differences separately, so I'll throw in a fix for that.